twindle-cli/env.js

```diff
@@ -1,13 +1,15 @@
+<<<<<<< HEAD:twindle-cli/twitter/utils/twitter-api-keys.js
+const { exit } = require("yargs");
+=======
 //  possibly include env related errors in a single location here
+>>>>>>> a9fd6f77e2de907238266312f792ea0050b70234:twindle-cli/env.js
```

Comment:
This is not correct. I think there were some merge conflicts which have not been resolved properly and the code was committed including these changes.